fmriprep/workflows/bold/resampling.py

```diff
@@ -272,11 +279,9 @@ def init_bold_std_trans_wf(
 
     """
     workflow = Workflow(name=name)
-    output_references = [s.fullname for s in spaces.snapshot if s.dim == 3 and s.standard]
-
-    std_spaces_set = {s for s in spaces.get_std_spaces(dim=(3,)) if s}
+    output_references = spaces.cached.get_spaces(nonstandard=False, dim=(3,))
     std_vol_references = [
-        (s.fullname, s.spec) for s in spaces.spaces if s.fullname in std_spaces_set
+        (s.fullname, s.spec) for s in spaces.references if s.standard and s.dim == 3
```

Comment:
wouldn't this be equivalent to:
```suggestion
        (s.fullname, s.spec) for s in spaces.references.get_standard(dim=(3,))
```

Reply:
it's an extra iteration